pkg/spammer/spammer.go

```diff
@@ -70,7 +70,7 @@ func (s *Spammer) DoSpam(shutdownSignal <-chan struct{}) (time.Duration, time.Du
 	iotaMsg := &iotago.Message{
 		NetworkID: s.networkID,
 		Parents:   tips.ToSliceOfArrays(),
-		Payload:   &iotago.Indexation{Index: indexation, Data: []byte(messageString)},
+		Payload:   &iotago.Indexation{Index: []byte(indexation), Data: []byte(messageString)},
```

Comment:
Should we encode the index as UTF8 here, because we are "the client lib" here somehow :)

Reply:
wdym encode as utf-8? indexation is already a string which unless users do something weird it should be a valid utf-8 string